Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -6449,3 +6449,57 @@ def to_display(self):
             'total': len(self.data),
             'data': self.data
         })
+
+
+class IndicatorsSearcher:
```

Comment:
What is the reason to use a class and not a simple function?

Reply:
We call this function couple of times and we want to keep some information between those calls, such as the page number or the `searchAfter` value.